### PR TITLE
Add some documentation about extraction

### LINKED ISSUE #228: Inherit existing translations
When i set up a symfony project i like to use existing bundles (fos user, knp paginator eq). But when i import translations the existing translations extracted are empty.

```xml
<segment>
  <source>resetting.check_email</source>
  <target></target>
</segment>
```

Can the extractor to be set up to extract the translations, and when a translation exists in the bundle we include the base translation on extract?

```xml
<segment>
  <source>resetting.check_email</source>
  <target>An email has been sent to %email%. It contains an activation link you must click to activate your account.</target>
</segment>
```

Or am i missing a flag that does just this?

### PR DESCRIPTION
**Pull request by odolbeau**
Add some documentation regarding extraction.

Fix https://github.com/php-translation/symfony-bundle/issues/228

### CODE REVIEW

File: symfony/extracting-translations.rst [hunk 1]
```diff
@@ -1,4 +1,38 @@
...
+    translation:
+      locales: ["en", "fr", "sv"]
+      configs:
+        app:
+          dirs: ["%kernel.root_dir%/Resources/views", "%kernel.root_dir%/../src"]
+          output_dir: "%kernel.root_dir%/Resources/translations"
+          excluded_names: ["*TestCase.php", "*Test.php"]
+          excluded_dirs: [cache, data, logs]
+
+With the configuration above you may extract all translation key from your source
```
Comment (bocharsky-bw) [inline symfony/extracting-translations.rst]: `... all translation keys...`

**Review by bocharsky-bw — APPROVED** [review]
Just a minor misprint tweak, but this is a great improvement for this chapter 👍

File: symfony/index.rst [hunk 2]
```diff
@@ -62,8 +62,8 @@ Below is an example of configuration that is great to start with.
           excluded_names: ["*TestCase.php", "*Test.php"]
           excluded_dirs: [cache, data, logs]
 
-With the configuration above you may extract all translation key from your source
-code by running
+With the configuration above you may extract all translation keys from your
```
Comment (odolbeau) [inline symfony/index.rst:65]: @bocharsky-bw I also corrected this file according to your comment as I copy pasted the first part of the symfony / extraction documentation from here.

Comment (bocharsky-bw) [reply symfony/index.rst:65]: Thanks 👍

**Comment by bocharsky-bw** [comment]
Thank you, Olivier, for adding missing docs to this section!

**Pull request merged**